transport/src/test/java/io/netty/channel/pool/FixedChannelPoolTest.java

```diff
@@ -300,6 +300,35 @@ public void run() {
         channel.close().syncUninterruptibly();
     }
 
+    @Test
+    public void testReleaseClosed() {
+        EventLoopGroup group = new LocalEventLoopGroup(1);
```

Comment:
shouldn't we be closing this at the end of the test? consider creating this in a `@BeforeClass` and cleaning up in `@AfterClass`.